spec/support/helpers.rb

```diff
@@ -24,6 +24,10 @@ def stub_omniauth(user: nil, provider: nil)
     # TODO: Move this to be returned with the user.
     stub_api_v2_request(
       "/recruitment_cycles/#{Settings.current_cycle}/providers",
+      resource_list_to_jsonapi([provider], meta: { count: 1 }),
+    )
+    stub_api_v2_request(
+      "/recruitment_cycles/#{Settings.current_cycle}/providers?page[page]=1&page[per_page]=1",
```

Comment:
something in our tests is making an API call to request for a single provider as this endpoint